src/containers/FilmFrontpage/AboutNdlaFilm.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  BaseArticle,
+  TransformedBaseArticle,
+  transformArticle,
+} from '../../util/transformArticle';
+import { GQLArticle_ArticleFragment } from '../../graphqlTypes';
+
+const StyledAside = styled.aside`
+  background: ${colors.brand.dark};
+  color: ${colors.white};
+  display: flex;
+  padding: ${spacing.normal} ${spacing.normal} ${spacing.medium};
+  > div {
+    padding: ${spacing.normal};
+    width: 50%;
+    h2 {
+      text-transform: uppercase;
+      letter-spacing: 0.05em;
+      color: #fff;
+      margin: 0 0 ${spacing.small} 0;
+    }
+  }
+  button {
+    color: #fff;
+    &:hover,
+    &:focus {
+      color: ${colors.brand.light};
+    }
+  }
+  ${mq.range({ until: breakpoints.tablet })} {
+    flex-direction: column;
+    > div {
+      width: auto;
+      &:first-of-type {
+        padding-bottom: 0;
+      }
+    }
+  }
+`;
+
+const StylediFrame = styled.iframe`
+  height: 100%;
+  width: 100%;
+  border: 0;
+  margin: 0;
+  padding: 0;
+`;
+
+const StyledModalBody = styled(ModalBody)`
+  h2 {
+    margin: 0;
+  }
+`;
+
+interface VisualElementProps {
+  visualElement: {
+    alt?: string;
+    url: string;
+    type: string;
+  };
+}
+
+const VisualElement = ({ visualElement }: VisualElementProps) => {
+  const { type, url, alt } = visualElement;
+  if (type === 'image') {
+    return <Image src={url} alt={alt ?? ''} />;
+  } else if (type === 'brightcove') {
+    return <StylediFrame allowFullScreen={true} src={url} />;
+  } else {
+    return null;
+  }
+};
+
+interface AboutNdlaFilmProps {
+  aboutNDLAVideo: {
+    title: string;
+    description: string;
+    visualElement: {
+      alt?: string;
+      url: string;
+      type: string;
+    };
+  };
+  article?: BaseArticle;
+}
+
+const AboutNdlaFilm = ({ aboutNDLAVideo, article }: AboutNdlaFilmProps) => {
+  const { t, i18n } = useTranslation();
+  const titleId = 'about-ndla-film-title';
+
+  const iArticle = useMemo(() => {
+    if (article) {
+      return transformArticle(
+        article,
+        i18n.language,
+      ) as TransformedBaseArticle<GQLArticle_ArticleFragment>;
+    }
+    return undefined;
+  }, [article, i18n.language]);
+
+  return (
+    <div className="o-wrapper">
+      <StyledAside aria-labelledby={titleId}>
+        <div>
+          <VisualElement visualElement={aboutNDLAVideo.visualElement} />
+        </div>
+        <div>
+          <Heading element="h2" headingStyle="h2" id={titleId}>
+            {aboutNDLAVideo.title}
+          </Heading>
+          <Text element="p">{aboutNDLAVideo.description}</Text>
+          {iArticle && (
+            <Modal>
+              <ModalTrigger>
+                <Button variant="link">{t('ndlaFilm.about.more')}</Button>
+              </ModalTrigger>
+              <ModalContent size="full">
+                <ModalHeader>
+                  <ModalCloseButton />
+                </ModalHeader>
+                <StyledModalBody>
+                  <Article article={iArticle} label={''} />
```

Comment:
Kunne hatt denne slik, men berre pirk.
```suggestion
                  <Article article={iArticle} label="" />
```